workers/base.py

```diff
@@ -8,11 +8,19 @@ def __init__(self, command_timeout, **args):
   def _setup(self):
     return
 
-  def format_id(self, *args, separator='/'):
-    return separator.join(["bt-mqtt-gateway", repr(self), *args])
+  def format_discovery_topic(self, mac, *sensor_args):
+    node_id = '0x{}'.format(mac.replace(':', ''))
```

Comment:
is it really needed? i think the mac as the node_id should be ok, or not (node_id = mac)?
i'm not sure if its ok for mqtt thinking if not, i think node_id = mac.replace(':', '-') is a more readable solution.

Reply:
mqtt allows for either `-` or `:`.The idea was to keep it short. But we might just use node_id = mac, so: `my_discovery_prefix/sensor/4C:65:A8:DA:44:A2/mithermometer_wohnen_temperature` as an example.

Reply:
I use full mac addresses in my implementations with MQTT as a "node" identifier.  Easy to understand what you are looking at and intuitive for those who are familiar with MAC addresses and what they mean for BLE.

Reply:
I agree, @ronytomen.